src/mcare_backend/mcare_backend/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for mcare_backend project.
+
+Generated by 'django-admin startproject' using Django 2.2.12.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+with open("C:\my_songs\Stuffs On Git\buildForSdgs\passwords.txt") as f:
+    SECRET_KEY = f.read().strip()
```

Comment:
Kindly follow guide on line 15 to make reference to files that are used in the project.